src/alto-ui/Aside/__tests__/Aside.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Aside from '../Aside';
+import CloseButton from '../../CloseButton';
+
+jest.mock('../../CloseButton', () => {
+  return () => {
+    return <div className="CloseButton" />;
+  };
+});
+
+describe('<Aside />', () => {
+
+  const getWrapper = props => {
+    console.log(props);
+    return mount(<Aside {...props} />);
+  };
+
+  it('should return null if show props is falsy', () => {
```

Comment:
```suggestion
  it('should return null if show props are falsy', () => {
```

Reply:
Changed to it('should no render if show `prop is` falsy', () => {